pkg/operator/handler.go

```diff
@@ -123,16 +200,19 @@ func conditionResourceApply(cr *regopapi.ImageRegistry, status operatorapi.Condi
 	}
 }
 
-func conditionDeployment(cr *regopapi.ImageRegistry, status operatorapi.ConditionStatus, m string, modified *bool) {
-	if status == operatorapi.ConditionFalse {
+func conditionSyncDeployment(cr *regopapi.ImageRegistry, syncSuccessful operatorapi.ConditionStatus, m string, modified *bool) {
+	reason := "DeploymentProgressed"
+
+	if syncSuccessful == operatorapi.ConditionFalse {
+		reason = "DeploymentInProgress"
 		logrus.Errorf("condition failed on %s %s/%s: %s", cr.GetObjectKind().GroupVersionKind().Kind, cr.Namespace, cr.Name, m)
```

Comment:
this should log what condition is being set to failed.  Also is this really an error we should always be logging?  We would expect this in cases where the deployment is still progressing.

Reply:
@bparees Removed.